Riot/Utils/EventFormatter.m

```diff
@@ -64,6 +64,28 @@ - (void)initDateTimeFormatters
 }
 
 - (NSAttributedString *)attributedStringFromEvent:(MXEvent *)event withRoomState:(MXRoomState *)roomState error:(MXKEventFormatterError *)error
+{
+    NSAttributedString *string = [self unsafeAttributedStringFromEvent:event withRoomState:roomState error:error];
+    
+    // If we cannot create attributed string, but the message is nevertheless meant for display (e.g. not an edit event), show generic error
+    // instead of a missing message on a timeline.
+    if (
+        (!string || string.length == 0)
+        && [self.eventTypesFilterForMessages containsObject:event.type]
+        && !event.isEditEvent
+    )
+    {
+        MXLogError(@"[EventFormatter]: Cannot format string for displayable event: %@, error: %lu", event.eventId, *error);
```

Comment:
Can we log more things to find the issue more quickly? Like:
- is `roomState` different of nil ?
- `roomState.membersCount.members``
- `event.type`
- `event.content[@"msgtype"] `

Reply:
Makes sense, added